plugins/gpio/GPIODriver.cpp

```diff
@@ -144,9 +153,19 @@ bool GPIODriver::SetupGPIO() {
       failed = true;
       break;
     }
-    if (write(fd, direction.c_str(), direction.size()) < 0) {
-      OLA_WARN << "Failed to enable output on " << str.str() << " : "
-               << strerror(errno);
+    // Assign correct initial state
+    if (inverted) {
+        direction = &inverted_direction;
+    } else {
+	direction = &normal_direction;
+    }
+
+    OLA_DEBUG << "Configuring GPIO pin " << static_cast<int>(*iter)
+	      << " with " << ( inverted ? "inverted" : "normal" ) << " logic";
+    if (write(fd, direction->c_str(), direction->size()) < 0) {
+      OLA_WARN << "Failed to enable output on " << str.str()
+	       << " with " << ( inverted ? "inverted" : "normal" ) << " logic"
+	       << " : " << strerror(errno);
```

Comment:
I think this should fix the outstanding lint issues @nkaminski :
```suggestion
    OLA_DEBUG << "Configuring GPIO pin " << static_cast<int>(*iter)
              << " with " << (inverted ? "inverted" : "normal") << " logic";
    if (write(fd, direction->c_str(), direction->size()) < 0) {
      OLA_WARN << "Failed to enable output on " << str.str()
               << " with " << (inverted ? "inverted" : "normal") << " logic"
               << " : " << strerror(errno);
```